tests/library/dbg/tests/test_command_canary.py

```diff
@@ -44,6 +46,7 @@ async def test_command_canary(ctrl: Controller, binary: str, reg_name: str, skip
         await ctrl.step_instruction()
 
     canary_value, at_random = pwndbg.commands.canary.canary_value()
+    assert at_random is not None
```

Comment:
Why do we need the changes in this file? The test failed on Arch but didn't on other setups? Why?

Reply:
LSP? Or some linter[s] we do have?